### PR TITLE
Canvas ux qa updates

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA: see attached bugs

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3847/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3848/
Test PASSed.

**Comment by anthonydillon** [comment]
Code and QA is good :+1:. Might be nice in the future to animate the pan but that is a call for UX.

**Comment by kadams54** [comment]
LOL at the branch name.

**Comment by kadams54** [comment]
QA notes: zooming in and out worked fine, as did clicking to focus on a service. When I add a bundle it still shows off-screen until I click on the "Deploy changes" button.

**Comment by kadams54** [comment]
Code is :+1:.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3855/
Test FAILed.

File: jujugui/static/gui/src/app/views/topology/service.js [hunk 1]
```diff
@@ -1378,11 +1394,6 @@ YUI.add('juju-topology-service', function(Y) {
           });
         }
       }
-      // Find the centroid of our hull of services and inform the
-      // topology.
-      if (vertices.length) {
-        this.findCentroid(vertices);
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/views/topology/service.js]: Do we ever want to pan to a service when it gets updated? This fixes the issue where services were panned to during deploy.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3856/
Test FAILed.

**Comment by kadams54** [comment]
QA is :ok:. @huwshimi is going to address the issue with bundles that have only a single charm (e.g., mongodb cluster) is a separate card and PR.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3862/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3863/
Test PASSed.

**Comment by huwshimi** [comment]
Thanks for the reviews! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**